style.css

```diff
@@ -0,0 +1,69 @@
+    header {
+        position: fixed;
```

Comment:
this property should be accompanied with as minimum 2 properties from next 4 (top, left, right, bottom)

Reply:
That's fixed!

Reply:
that is not fixed yet. Please, be more attentive